OpenRA.Mods.Common/Traits/SupportPowers/ParatroopersPower.cs

```diff
@@ -80,31 +73,25 @@ public Actor[] SendParatroopers(Actor self, WPos target, bool randomize = true,
 			if (!self.World.Map.Rules.Actors.TryGetValue(utLower, out unitType))
 				throw new YamlException("Actors ruleset does not include the entry '{0}'".F(utLower));
 
+			// Cache target before aircraft altitude is added to spawn camera on ground
```

Comment:
Im confused by this, I thought the camera was removed o.O

Reply:
Unlucky wording, 'camera' should really be 'camera effect' or 'camera reveal'.
Basically, without this the shroud would be revealed around the aircrafts screen position (in other words, a few cells north of the target cell).

Reply:
Adding just in case: only the camera *actor* spawning was removed, all support powers now support revealing via effect instead.